libphx/src/Lua.cpp

```diff
@@ -276,8 +276,8 @@ void Lua_Backtrace () {
       if (!funcName)                 funcName = "<null>";
 
       cstr stackFrame = (line > 0)
-        ? StrFormat("  %s(%i): %s", fileName, line, funcName)
-        : StrFormat("  %s: %s", fileName, funcName);
+        ? StrFormat("  #%i %s at %s:%i", iStack, funcName, fileName, line)
```

Comment:
This is a stacktrace formatting change that matches other debuggers like gdb/lldb. It allows the Lua file:line to be clickable in most terminals.